packages/gasket-plugin-analyze/lib/index.d.ts

```diff
@@ -10,9 +10,6 @@ declare module '@gasket/core' {
   }
 }
 
-const plugin: Plugin = {
-  name: '@gasket/plugin-analyze',
-  hooks: {}
-};
+declare const plugin: Plugin;
 
-export = plugin;
+export default plugin;
```

Comment:
You have several plugin type changing this to default exports. However, we've had type issues because these packages are not `type: module`. What's the reason for the change now and have you vetted the imports with a TypeScript app?

Reply:
I am seeing these errors <img width="706" alt="Screenshot 2025-03-04 at 10 59 01 AM" src="https://github.com/user-attachments/assets/55d66042-8d1f-41ba-be65-7af7c0b15e66" />

Reply:
Ok - It indeed reads better. Well, if it works, it works.